luigi/parameter.py

```diff
@@ -497,6 +506,19 @@ def normalize(self, value):
         return datetime.date(year=value.year - delta, month=1, day=1)
 
 
+class PastDateParameter(DateParameter):
+    """
+    caveat: if your host default timezone is different from your `business` timezone
+    that might not work for you
+    """
+
+    def normalize(self, value):
```

Comment:
by overriding the parent normalize, you lose the assurance that the param is a date and not a datetime, etc.

Reply:
I believe this is resolved?

Reply:
@honnix yes